thunder/torch/__init__.py

```diff
@@ -184,6 +184,13 @@ def is_cuda(a: TensorLike, /) -> bool:
     return a.device.devicetype is devices.DeviceType.CUDA
 
 
+# is nested always returns False for now:
+# https://github.com/Lightning-AI/lightning-thunder/issues/93#issuecomment-2030416883
+@torchsymbol(torch.Tensor.is_nested, is_property=True, id="torch.is_nested")
```

Comment:
fyi @k223kim, thanks to your work it's easy to add more properties!

Reply:
Happy to help! 🚀